bin/node-template/runtime/src/lib.rs

```diff
@@ -100,58 +105,6 @@ pub mod impls {
 		}
 	}
 
-	/// Update the given multiplier based on the following formula
```

Comment:
Did substrate have multiplier migration, changing from -1 to 0?

Reply:
It's OK with Crab.

https://github.com/paritytech/substrate/pull/6334/files#diff-c624291b4dc7bad33847a05c04bdb83eR255